### DISCUSSION TITLE
Daily Updates

### DISCUSSION THREAD

**Discussion by godsakani** [100-Days-of-Coding/Coding-for-100-days#24] [General]
Hello @everyone 
Trust we are doing great. 
This section is created to enhance our daily updates:
Meaning what is expected here is:
1) What you did that day;
2) What were your impediments
3) What you plan to do

**Comment by ndongchrist** [comment]
Waoooh really interesting. 
I don't really know what to do. If you could guide us with small small tasks, daily tasks. 
For us to report what we did that day.

**Reply by PetraAG** [reply]
You could check out the Read me file... I'm sure there will be tasks in the course we are following

**Comment by godsakani** [comment]
Hello @everyone 
happy holiday lets not forget to give our updates.

for me what i work on today was 
Doing some updates on my ide,
Did small revision on dart.
started with some basic widget such as appbar, scaffold, floating action button, center.

![des](https://user-images.githubusercontent.com/70459251/166332891-e422f6f7-9689-45e6-b2ac-b25c8d4abae0.png)

**Comment by PetraAG** [comment]
Daily updates from me
so, i checked out the sites i may like folder in the course, and cool...freeCourseSite is cool, you can literally get all courses for free.
Then the courseclub and forum.freecoursesite was giving me error code 522, i don't know if it is my browser alone
I started with module 1: introduction, downloaded the course outline 
[App Brewery Flutter Course Syllabus.pdf](https://github.com/100-Days-of-Coding/Coding-for-100-days/files/8607821/App.Brewery.Flutter.Course.Syllabus.pdf)
, and finish the introduction of the course. Why flutter, what is flutter etc. Interesting stuff such as one codebase, constraints, hot reload etc. oh, and widgets. Excited to finish this journey

**Comment by PetraAG** [comment]
joined the discord channel as well

**Reply by godsakani** [reply]
So did you faced any impediments

**Reply by PetraAG** [reply]
Nope... Since i have my system ready, i will simply browse through installation just to pick up one or two things, and officially get hands on with module 3...🤗

**Reply by godsakani** [reply]
> Nope... Since i have my system ready, i will simply browse through installation just to pick up one or two things, and officially get hands on with module 3...🤗

cool

**Comment by godsakani** [comment]
Hello everyone.
Hope we are doing well.

Just wish to say no matter the tutorial you are following if is that you have to go through the intro of a course  go through course they new features you can grap.

**Comment by godsakani** [comment]
Hello @everyone trust we are doing well.
Concerning what I did today was working on image assets, adding images and also how to structure my yaml file.
Also worked on how to change app Icon both for android and IOS

**Reply by ndongchrist** [reply]
honestly since i didnot start any thing.
i still have other projects dtill going on my hands. 
But i will try my best to always do something

**Reply by godsakani** [reply]
> honestly since i didnot start any thing. i still have other projects dtill going on my hands. But i will try my best to always do something

ok Cool always try to give your daily updates

**Comment by godsakani** [comment]
Hi everyone 
 trust we are doing ok.

For today's updates:
Studied some widgets:
single-child widget
1. Container widget
Multi-child widget
1. Column widget
3. Row widget

With properties

**Comment by godsakani** [comment]
Hello, @everyone I noticed we have been quiet. 
so far these past days have been working on widgets 
and some hands-on design with the widgets and happy is moving well.

My greatest joy will be if we go through this together.
I can say somewhat that being able to identify parts when it comes to design I can do

**Comment by PetraAG** [comment]
Hi... I am following though slowly... Great job. Keep up the good work

**Comment by godsakani** [comment]
Good evening everyone. 
what did to was to push all this week activities to git you check the branch week-one. it contains all the task done this week.

**Comment by PetraAG** [comment]
Good morning..
For this week, i learnt so many interesting concepts


The layout of a flutter app... MaterialApp, scaffold (appBar,
backgroundColor etc) learnt how to use the flutter documentation, did the I
am rich app, and I am poor challenge, learnt how to manipulate images in
flutter, i also learnt about using icons (even designing mine on canva)
etc. Out of the box, got tips on how to avoid distractions, and also how to
be a better codder by scheduling...

**Comment by godsakani** [comment]
Good evening hope we are doing great.

Today worked on some widget which helps in responsiveness
which are 
Expanded with it properties
flatbutton
Images

**Comment by godsakani** [comment]
Hello everyone 
Hope we are doing good 
Today worked on
 dart data Types,
 dart functions
Stateful and Stateless widget

**Comment by godsakani** [comment]
Good evening everyone
 My today work :

stateful widget
stateless widget
math functions

**Comment by godsakani** [comment]
Hello @everyone

Day_09 100daysOfCode with flutter and dart.
Stateless widget
Stateful widget
Dart functions
Dart data types
Alert widget

Small dicegame with success notification
